flixel/graphics/FlxGraphic.hx

```diff
@@ -514,6 +514,8 @@ class FlxGraphic implements IFlxDestroyable
 	 */
 	public inline function getFramesCollections(type:FlxFrameCollectionType):Array<Dynamic>
 	{
+		if (this.isDestroyed) return [];
+
```

Comment:
```suggestion
		if (this.isDestroyed)
		{
			FlxG.log.warn('Invalid call to getFramesCollections on a destroyed graphic');
			return [];
		}
		
```

Reply:
Seems good! I think I may have had a warning in a previous iteration of this.